### DISCUSSION TITLE
How to change snake glyph/emoji?🐍

### DISCUSSION THREAD

**Discussion by ammnt** [starship/starship#2515] [Q&A] [answered]
Hi, guys!

Thanks for the best minimalistic prompt tool😘Please, explain how to change snake glyph/emoji:

![image](https://user-images.githubusercontent.com/45385632/112661239-5d1eb000-8e67-11eb-9f8c-5962e8a583e3.png)

macOS 11.2.3
starship 0.51.0
branch:
commit_hash:
build_time:2021-03-23 17:59:38
build_env:rustc 1.50.0,
zsh 5.8 (x86_64-apple-darwin20.1.0)

Cheers🤟

**Comment by vladimyr** [comment, marked as answer]
@ammnt You need to change the `symbol` option of the `python` module. Check this example: https://starship.rs/config/#example-40

**Comment by ammnt** [comment]
@vladimyr, i tried it before, but no success. Works now fine!! Thank you & Спасибо👍🏻

**Reply by vladimyr** [reply]
> Thank you & Спасибо👍🏻

Nema na čemu, това́рищ. Greetings from Croatia and thanks for the chuckle :grinning:
